libraries/Microsoft.Bot.Builder/BotState.cs

```diff
@@ -304,13 +298,19 @@ public Task DeleteAsync(ITurnContext turnContext)
             /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
             public async Task<T> GetAsync(ITurnContext turnContext)
             {
+                await _botState.LoadAsync(turnContext).ConfigureAwait(false);
                 try
                 {
                     return await _botState.GetPropertyValueAsync<T>(turnContext, Name).ConfigureAwait(false);
                 }
                 catch (KeyNotFoundException)
```

Comment:
>KeyNotFoundException [](start = 23, length = 20)

I thought this would be changed to MissingMemberException

Reply:
This is a change steve does on the js side, they auto load